### PR TITLE
feat: Add drawable suggesting editable profile picture in editprofileframent.

### LINKED ISSUE #963: Add camera icon on Profile image in Edit Profile Fragment.
**Describe the feature you'd like**

A small camera/edit icon on the bottom left corner of the profile picture to let users know that's where the can upload their profile picture. 

**Screenshots**

Sample
edit
This will be over existing profile image.

**Additional context**

@liveHarshit Your views?

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #963 

changes: Changed default profile picture so as to remove padding in picture. Added semicircle on profile picture in editprofilefragment. Tried adding pencil, but it didn't look very good, so went with this instead.

Screenshots for the change:
image

image

### CODE REVIEW

**Comment by angmas1** [comment]
> Looks good. Please resolve conflicts and one suggestion would be once the user has updated his/her profile picture, the update text can be hidden since the user knows that he can update his picture

Working on it. 👍

**Comment by angmas1** [comment]
@nikit19 added the feature you suggested.

**Comment by angmas1** [comment]
ezgif com-video-to-gif

**Comment by angmas1** [comment]
@iamareebjamal @nikit19 Please review.

**Comment by angmas1** [comment]
Okay

**Comment by angmas1** [comment]
@iamareebjamal please review.

**Comment by iamareebjamal** [comment]
How about a small edit FAB instead

**Comment by angmas1** [comment]
okay, working on that.

**Comment by angmas1** [comment]
image

**Comment by iamareebjamal** [comment]
How about edit icon. Camera seems too specific

**Comment by angmas1** [comment]
okay, replacing

File: app/src/main/res/drawable/ic_account_circle_grey.xml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="56dp"
-    android:height="56dp"
-    android:viewportHeight="24.0"
-    android:viewportWidth="24.0">
+    android:viewportWidth="400"
```
Comment (iamareebjamal) [inline app/src/main/res/drawable/ic_account_circle_grey.xml]: Revert

File: app/src/main/res/drawable/ic_edit_white.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```
Comment (iamareebjamal) [inline app/src/main/res/drawable/ic_edit_white.xml]: Just ic_edit

Comment (angmas1) [reply app/src/main/res/drawable/ic_edit_white.xml]: Okay

**Comment by angmas1** [comment]
@iamareebjamal please review.

**Pull request merged**